R/pkg/R/DataFrame.R

```diff
@@ -59,6 +60,7 @@ setMethod("initialize", "SparkDataFrame", function(.Object, sdf, isCached) {
 #' @export
 #' @param sdf A Java object reference to the backing Scala DataFrame
 #' @param isCached TRUE if the SparkDataFrame is cached
+#' @note dataFrame since 2.0.0
```

Comment:
you might not need to call this out with a separate note btw, this is an internal function to help wrap the returned Java object "handle" to a R class, not intend to be called by users.